src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -259,7 +292,7 @@ class TimelineMarkerCanvas extends PureComponent {
     const minDuration =
       rangeLength *
       viewportLength *
-      (rowHeight * 2 * MARKER_DOT_RADIUS / containerWidth);
+      (rowHeight * 2 * MARKER_MIN_DOT_RADIUS / containerWidth);
```

Comment:
I tried with the previous value, and I definitely prefer the behavior with this value instead.